Formula/qdmr.rb

```diff
@@ -0,0 +1,34 @@
+class Qdmr < Formula
+  desc "Codeplug programming tool for DMR radios"
+  homepage "https://dm3mat.darc.de/qdmr/"
+  url "https://github.com/hmatuschek/qdmr/archive/refs/tags/v0.11.2.tar.gz"
+  sha256 "812e51ac9e2c4fe430673d8f7c9a2f351feb5f70275755c6da88e26bbab2b272"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "libusb"
+  depends_on "qt@5"
```

Comment:
Does it work with the newest QT?

Reply:
Not without changes to the upstream projects CMakeLists.txt.

Reply:
Yeah, I meant 6.